### PR TITLE
Seperate button for editing headshot. 

### PR DESCRIPTION
**Pull request by kanchan-bluebash**

### CODE REVIEW

File: app/views/shared/_modal_toggle.js.erb [hunk 1]
```diff
@@ -2,3 +2,7 @@ if (!$('<%= id %>').length) {
   $('main').append("<%= yield %>");
 }
 $('<%= id %>').modal('toggle');
+
+$('#edit_cv_headshot').on('hidden.bs.modal', function () {
+  window.location.reload();
+});
```
Comment (darmenise) [inline app/views/shared/_modal_toggle.js.erb:8]: Please convert to simple javascript

File: app/views/cvs/edit_headshot.js.erb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+$("#profile_img").on('click', function(e){
+  if (confirm("Are you sure?")) {
+    $("#cv_remove_headshot").val('1')
+    $('.preview').addClass('d-none').html('')
+    $('.default-preview').removeClass('d-none')
+    this.classList.add('d-none')
+  }
+});
```
[the remark this answers is not in the document]
Comment (kanchan-bluebash) [reply app/views/cvs/edit_headshot.js.erb:38]: @darmenise Actually, I have just moved the existing code from `app/views/cvs/edit_intro.js.erb` here. Other code in this file is also in jQuery. If this needs to be in plain javascript please create another issue for this. I'll check for its conversion.

File: app/views/shared/_modal_toggle.js.erb (hunk 1 above)
Comment (kanchan-bluebash) [reply app/views/shared/_modal_toggle.js.erb:8]: @darmenise i have used bootstrap modal event here which is in jQuery.

Comment (vishal-bluebash) [reply app/views/shared/_modal_toggle.js.erb:8]: @darmenise  we can not fix it, for now as it is bootstrap dependency,  One we migrate to new version of bootstrap we can get rid of it (in new bootstrap they are removing jquery dependency)

**Comment by kanchan-bluebash** [comment]
**Demo:-** Seperate icon for headshot.

Screenshot from 2021-07-14 19-08-11

**Review by vishal-bluebash — APPROVED** [review]

**Pull request merged**